docs/_partner_api/errors.md

```diff
@@ -9,13 +9,14 @@ redirect_from: /partner_api/
 
 Zapier uses HTTP response codes to indicate the success or failure of an API request.
 
-| Code    | Status         | Explanation                                                     |
-| ------- | -------------- | --------------------------------------------------------------- |
-| **200** | OK             | Successful request.                                             |
-| **400** | Bad Request    | Invalid request, or invalid parameters .                        |
-| **403** | Authentication | Not authorized.                                                 |
-| **404** | Not Found      | The resource requested was not found.                           |
-| **5xx** | Server Error   | A fatal error occurred while processing the request. Try again. |
+| Code    | Status            | Explanation                                                                |
+| ------- | ----------------- | -------------------------------------------------------------------------- |
+| **200** | OK                | Successful request.                                                        |
+| **400** | Bad Request       | Invalid request, or invalid parameters .                                   |
+| **403** | Authentication    | Not authorized.                                                            |
+| **404** | Not Found         | The resource requested was not found.                                      |
+| **429** | Too Many Requests | Too many requests were performed within a window of time. Try again later. |
```

Comment:
You might need another row of `|` under this one?

Reply:
I'm having trouble following what the expectation here would be.

If I use GitHub to display the rich diff I get the following as-is https://cdn.zappy.app/52fed74fcac65e0629fdb47a15128aad.png

Would you either expect that to render differently once published? Or does the rich diff need altering?

Reply:
🤔 I think I misread the wrapped lines as newlines in the diff. Disregard then.